script/RumpelConfig.sol

```diff
@@ -72,6 +74,9 @@ library RumpelConfig {
     address public constant MAINNET_FLUID_VAULT_SUSDE_USDC = 0x3996464c0fCCa8183e13ea5E5e74375e2c8744Dd;
     address public constant MAINNET_FLUID_VAULT_SUSDE_USDT = 0xBc345229C1b52e4c30530C614BB487323BA38Da5;
     address public constant MAINNET_FLUID_VAULT_SUSDE_GHO = 0x2F3780e21cAba1bEdFB24E37C97917def304dFFA;
+    address public constant MAINNET_FLUID_VAULT_SUSDE_USDT_USDT = 0x7503b58Bb29937e7E2980f70D3FD021B7ebeA6d0;
```

Comment:
Can we have a different naming convention or comments to document which assets are smart collateral vs smart debt (or both)?

Reply:
![Screenshot 2025-02-27 at 11 36 19 AM](https://github.com/user-attachments/assets/4b7af675-6ebe-4bcd-8b2b-b7b233981661)
Updated to use their convention, with "DEX" in the name placed depending on if it applies to collat vs debt. and added some comments